### DISCUSSION TITLE
Plans for making SQL Server Open to non pro users

### DISCUSSION THREAD

**Discussion by fshaikh-caa** [ariga/atlas#3255] [Q&A] [answered]
Hello, 

Like the title asks, are there any plans for opening SQL Server to non pro users?

**Comment by a8m** [comment, marked as answer]
Hey @fshaikh-caa 👋 

The payments for the Pro tier are essential to keeping the company running, and there are no plans to change this in the foreseeable future.

You can apply for a [hacker-license](https://atlasgo.io/hacker-license) if you want to use Atlas Pro for non-commercial use.
